packages/batcher/runtime/src/index.ts

```diff
@@ -166,31 +171,79 @@ async function main(): Promise<void> {
     'Game Batcher', // TODO: it doesn't matter now, but there is no way for the batcher to get the name of the game
     await getRemoteBackendVersion()
   );
-  const provider = await getWalletWeb3AndAddress(ENV.CHAIN_URI, privateKey);
+  let provider: EthersEvmProvider | AvailJsProvider;
+  let batchedTransactionPoster;
+  let getCurrentBlock;
+
+  if (!ENV.BATCHER_AVAIL_LIGHT_CLIENT) {
```

Comment:
I've been actually wondering if it wouldn't make more sense to just have one "binary" per network instead of having a single one where some environment variables work for one and others for another.

Although maybe the issue it's just that environment variables are really clunky for configuration.

In this case what would make sense to have, a variable that can be either `evm | avail | mina` ?

Reply:
the executables are pretty large, so probably one exe per network isn't ideal filesize-wise